src/Annotation/Controller/Id.php

```diff
@@ -2,34 +2,28 @@
 
 namespace Apitte\Core\Annotation\Controller;
 
+use Doctrine\Common\Annotations\Annotation\NamedArgumentConstructor;
 use Doctrine\Common\Annotations\Annotation\Target;
 use Doctrine\Common\Annotations\AnnotationException;
 
 /**
  * @Annotation
  * @Target({"CLASS","METHOD"})
+ * @NamedArgumentConstructor()
  */
 final class Id
 {
 
 	/** @var string */
 	private $name;
 
-	/**
-	 * @param array<string, mixed|null> $values
-	 */
-	public function __construct(array $values)
+	public function __construct(string $name)
 	{
-		if (!isset($values['value'])) {
-			throw new AnnotationException('No @Id given');
-		}
```

Comment:
There checks are now replaced by native PHP and required arguments without default value in constructor :+1: